typescript/rest-nextjs-express/frontend/package.json

```diff
@@ -18,7 +18,8 @@
     "react-markdown": "^5.0.0"
   },
   "devDependencies": {
-    "@types/node": "14.14.28",
+    "prisma": "2.17.0",
```

Comment:
I think this dependency had been removed intentionally in this project.

Reply:
Ha, you're right! Thanks for catching this Jan!! 🦅 👁️  Fixing via: https://github.com/prisma/prisma-examples/pull/2577